CalibCalorimetry/EcalTPGTools/plugins/EcalTPGParamBuilder.h

```diff
@@ -1,6 +1,12 @@
 #ifndef ECALTPGPARAMBUILDER_H
 #define ECALTPGPARAMBUILDER_H
 
+//Author: Pascal Paganini - LLR
+//Date: 2006/07/10 15:58:06 $
+
+#define CMSSW_VERSION 340
```

Comment:
It is historical.  I'm not the author of this pgm.  I don't know if somewhere a version < 3_4_0 uses this condition.  Removing this definition and the tests after, works, from a compilation point a view.